### PR TITLE
docs: shows the runtimes in the right side toc menu

### PR DESCRIPTION
**Pull request by tnolet**
## Affected Components
* [ ] Content & Marketing
* [ ] Pricing
* [ ] Test
* [x] Docs
* [ ] Learn
* [ ] Other

## Pre-Requisites
* [ ] Code is linted (`npm run lint`)

## Notes for the Reviewer

- small QOL change. Lists runtime in toc

## Screenshots

CleanShot 2023-03-01 at 15 32 55

### CODE REVIEW

**Comment by clample** [comment]
Why weren't they showing up in the right sidebar before?

**Comment by tnolet** [comment]
> Why weren't they showing up in the right sidebar before?

@clample  the Hugo shortcode was not interpreted with `%` which means it's post-render which means it's not passed to the underlying page structure.

File: site/layouts/shortcodes/runtimes.html [hunk 1]
```diff
@@ -1,21 +1,18 @@
...
   {{ if eq .stage "DEPRECATED" }}
-  <p class="text-accent">This runtime will be available until {{ .runtimeEndOfLife }}. Please update your checks' runtimes and account settings to avoid failures due to automatic change to the next supported runtime.</p>
+    This runtime will be available until {{ .runtimeEndOfLife }}. Please update your checks' runtimes and account settings
+    to avoid failures due to automatic change to the next supported runtime.
+  {{ end }}
+  {{ .description }}
+
+
+  {{ range $key, $value := .dependencies }}
+  - [{{ $key }}](https://npmjs.com/package/{{ $key }}/v/{{ $value }}) {{ $value }}
```
Comment (clample) [inline site/layouts/shortcodes/runtimes.html:16]: Nit: Is it intentional to remove the target="_blank" and have the tab open in the same window?

Comment (tnolet) [reply site/layouts/shortcodes/runtimes.html:16]: @clample no such thing as `target="_blank"` in Markdown sadly. That's the tradeoff

Comment (clample) [reply site/layouts/shortcodes/runtimes.html:16]: Ah, ok. Thanks for clarifying.

**Review by clample — APPROVED** [review]

**Pull request merged**